base/strings/io.jl

```diff
@@ -173,6 +173,8 @@ julia> string("a", 1, true)
 """
 string(xs...) = print_to_string(xs...)
 
+string(s::Symbol) = unsafe_string(unsafe_convert(Ptr{UInt8}, s))
```

Comment:
```suggestion
string(s::Symbol) = String(s)
```

Reply:
Interesting: `String` shares the more-or-less same code, besides of being `@pure`.

```
In String(s) at strings/string.jl:82
>82  @pure String(s::Symbol) = unsafe_string(unsafe_convert(Ptr{UInt8}, s))
```